samcli/commands/init/init_templates.py

```diff
@@ -171,3 +184,79 @@ def is_dynamic_schemas_template(self, package_type, app_template, runtime, base_
             if option.get("appTemplate") == app_template:
                 return option.get("isDynamicTemplate", False)
         return False
+
+    def get_preprocessed_manifest(self, filter_value=None):
+        """
+        This method get the manifest cloned from the git repo and preprocessed it.
+        The structure of the manifest is shown below:
+
+        {
```

Comment:
Lets create a link to the actual manifest as well, we do not want two percieved sources of truth in the future in the form of a docstring and the actual contents within a different file. This can lead to drift.

Reply:
ok